### PR TITLE
Show error in case of client timeout in popup

### PR DESCRIPTION
**Pull request by Bettelstab**
Also simplifies reactivity in App.vue.

### CODE REVIEW

File: src/views/RequestError.vue [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<template>
+    <div class="container">
+        <SmallPage>
+            <Loader
+                state='error'
+                title='Invalid request'
+                message='Something went wrong with your request. Please try again.'
+                mainAction='Close'
+                @main-action='close'
```
Comment (danimoh) [inline src/views/RequestError.vue]: Use double quotes in HTML.

File: src/views/RequestError.vue [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            />
+        </SmallPage>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Watch, Vue } from 'vue-property-decorator';
```
Comment (danimoh) [inline src/views/RequestError.vue]: Unused imports `Prop` and `Watch`

File: src/views/RequestError.vue [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import { SmallPage } from '@nimiq/vue-components';
+import Loader from '../components/Loader.vue';
+
+@Component({components: {Loader, SmallPage}})
+export default class RequestError extends Vue {
+    private close() {
```
Comment (danimoh) [inline src/views/RequestError.vue]: Private methods should start with an underscore.

File: src/views/RequestError.vue [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        window.close();
```
Comment (danimoh) [inline src/views/RequestError.vue]: Should we just close here or reject the request?

File: src/lib/RpcApi.ts [hunk 5]
```diff
@@ -67,9 +67,15 @@ export default class RpcApi {
...
-        // Redirect to Safe (default) if there is no client.
-        // This happens if user clicks on a link to accounts.nimiq.com.
-        const onClientTimeout = () => { location.href = Config.redirectTarget; };
+        // If there is no request redirect to Safe (default) in redirect case,
+        // or show error in popup case.
+        const onClientTimeout = () => {
+            if (window.opener === null) {
+                location.href = Config.redirectTarget;
+            } else {
+                this._router.replace('/requestError');
```
Comment (sisou) [inline src/lib/RpcApi.ts]: camelCase in the the URL? I'd rather `/request-error`.

File: src/views/RequestError.vue [hunk 6]
```diff
@@ -0,0 +1,26 @@
+<template>
+    <div class="container">
+        <SmallPage>
+            <Loader
+                state='error'
+                title='Invalid request'
+                message='Something went wrong with your request. Please try again.'
```
Comment (danimoh) [inline src/views/RequestError.vue]: Should we be more specific?

Comment (Bettelstab) [reply src/views/RequestError.vue]: No :D

**Pull request merged**